wallet/wallet.go

```diff
@@ -182,3 +182,49 @@ func (w *Wallet) keyInfoForAddr(addr address.Address) (*types.KeyInfo, error) {
 	}
 	return info, nil
 }
+
+// Import adds the given keyinfos to the wallet
```

Comment:
seems like we should have a unit test or at least a daemon test for the round trip. if not i dont want to get in the way of more progress on killing nodeapi but probably good to start a list of things to circle back on once the api is killed...

Reply:
There is `TestWalletExportImportRoundTrip`